posthog/hogql_queries/insights/test/test_events_query.py

```diff
@@ -0,0 +1,105 @@
+from typing import Tuple, Any
+
+from freezegun import freeze_time
+
+from posthog.hogql_queries.events_query_runner import EventsQueryRunner
+from posthog.schema import (
+    EventsQuery,
+    EventPropertyFilter,
+    PropertyOperator,
+)
+from posthog.test.base import APIBaseTest, ClickhouseTestMixin, _create_event, _create_person
+
+
+class TestEventsQueryRunner(ClickhouseTestMixin, APIBaseTest):
```

Comment:
The changes in this PR are in mostly `property.py` and `parse_expr`, which is accessed from a lot of places. The tests are in `test_events_query.py`. 

Can you add these or some tests to `test_property.py` as well? Having this under `test_events_query.py` comes with the danger that someone might clean/refactor this file independently (e.g. when we move to a new runner system), and without that this is the only place where this logic is tested, remove them. 

However it seems like this broke some tests in `test_property.py`, as well...

Reply:
I missed those, they are updated now :)